### ISSUE TITLE
IconButton: display label when icon is not specified

### ISSUE THREAD

**Issue by rgtk** [sstur/react-rte#291]
`react-rte` [allows to use](https://github.com/sstur/react-rte/blob/9257c22e1c05c9a51148ee812e4995238bdfc8f6/src/ui/IconButton.js#L31) predefined set of icons only. For some cases, [available icons](https://github.com/sstur/react-rte/blob/6d8787a009d6574e27704521ce7caf94b2b1435a/src/ui/IconButton.css) are not enough to describe custom control.

I see two solutions:
1. Allow to pass custom webpack assets / URLs.
2. When `iconName` prop is not provided, display `label` as text so button won't be empty inside.

1 is too ambitious. 2 would do and is easy to implement.

**Comment by victorpavlenko** [comment]
@rgtk any progress?

**Comment by rgtk** [comment]
No. As I did not receive any input from author I didn't invest time for PR
and went with workaround.
